tests/extensions/inline/italic/full/valid/nested.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../basic.rng"?>
```

Comment:
there is the potential to be more clear in expressing the relationship between the sample and its schema if all samples for `basic.rng` were into a `.../full/basic/` folder. The `valid` and `invalid` could be nested into that, or transformed into prefix as the last level `.../full/basic/valid-nested.xml` to have a single view of both valid and invalid documents for a single schema.